adapter/internal/oasparser/envoyconf/routes_with_clusters.go

```diff
@@ -1244,6 +1244,79 @@ func CreateHealthEndpoint() *routev3.Route {
 	}
 	return &router
 }
+func CreateJwksEndpoint() *routev3.Route {
```

Comment:
I think we could improve the logic since this pattern is common across testkey endpoint and jwks endpoint in terms of route config.